apps/web/package.json

```diff
@@ -94,6 +94,7 @@
     "nodemailer": "^6.7.7",
     "otplib": "^12.0.1",
     "qrcode": "^1.5.1",
+    "raw-body": "^2.5.1",
```

Comment:
Used to calculate the signature from the request from Help Scout in its dedicated API